trecs/tests/test_ContentFiltering.py

```diff
@@ -75,12 +69,8 @@ def test_partial_arguments(self, items=None, users=None, attr=None):
         c = ContentFiltering(num_items=items)
         test_helpers.assert_correct_num_users(c.num_users, c, c.users_hat.shape[0])
         test_helpers.assert_correct_num_items(items, c, c.items_hat.shape[1])
-        test_helpers.assert_correct_size_generic(
-            c.num_attributes, c.num_attributes, c.items_hat.shape[0]
-        )
-        test_helpers.assert_correct_size_generic(
-            c.num_attributes, c.num_attributes, c.users_hat.shape[1]
-        )
+        # assert attributes look correct
+        assert c.users_hat.shape[1] == c.items_hat.shape[0]
```

Comment:
You can mostly ignore changes in this file, I was just adapting to changes in some of the input validation functions and thought I'd alter some tests here; I thought it was weird to pass `c.num_attributes` twice to `test_helpers.assert_correct_size_generic`